src/SPECS/hootenanny.spec.in

```diff
@@ -18,7 +18,8 @@ BuildRequires:  git glpk-devel graphviz libicu-devel
 BuildRequires:  hootenanny-words = 1.0.0
 BuildRequires:  libspatialite-devel
 BuildRequires:  log4cxx-devel nodejs-devel opencv-devel
-BuildRequires:  postgresql-devel >= 9.1
+#BuildRequires:  postgresql-devel >= 9.1
+BuildRequires:  postgresql93-devel
```

Comment:
Yes. I figured that Damian was already running 93 so it wouldn't be a problem.

I can go back to 92 and I have another strategy for avoiding the postgresql-libs 8.4 install that I'm going to test.